bin/eslint_d.js

```diff
@@ -31,7 +31,16 @@ if (cmd === 'start') {
   } else {
     var commands = ['stop', 'status', 'restart'];
     if (commands.indexOf(cmd) === -1) {
-      client.lint(process.argv.slice(2));
+      var concat = require('concat-stream');
```

Comment:
:+1:

Re: Using concat-stream, eslint uses the same dependency, so technically it's not any slower than eslint. It handles a lot of use cases, and if I'm honest, I'm not comfortable enough with streams to attempt to do it myself.